### DISCUSSION TITLE
git lfs Error opening .gitattributes file

### DISCUSSION THREAD

**Discussion by glhey** [git-lfs/git-lfs#4570] [Q&A] [answered]
git lfs track "<file pattern" eg \*.xl\* fails:
  "Error opening .gitattributes file"
It 'feels' like a file lock somewhere (eg Antivirus) but  .gitattributes is only Hidden, not Read Only, not System. Nothing in Windows Event Log.

If I kill the existing .gitattributes file and start afresh then 'git lfs track \"\*xl\*\"' succeeds; I can run git lfs track often with no problems and "git add/commit" with no problems.

A few hours later, and "git lfs track ... " fails. 

    "Error opening .gitattributes file"

I can edit .gitattributes manually with a text editor - no problem. 
Specs are: Win 10, Bitdefender, git version 2.32.0.windows.2, git-lfs/2.13.3 (GitHub; windows amd64; go 1.16.2; git a5e65851)
All very strange.

Thanks. 
GL

**Comment by bk2204** [comment, marked as answer]
Hey,

I'm not sure why this would be happening.  We don't typically see reports of this very often.

Typically we recommend that users always use Windows Defender, since certain other antivirus programs are known to break both Git and Git LFS, so you may try doing that instead and see if it solves your problem.

**Reply by glhey** [reply]
Thanks Brian, I deactivated Bitdefender firewall and activated Win Defender firewall. Random test on various repo's now succeeds. Not definitive, so I'll revisit if the problem reappears. Much obliged.
gl.
